web/sites/default/config/language/es/views.view.news_index.yml

```diff
@@ -141,77 +141,6 @@ display:
       link_url: null
       block_description: 'Noticias destacadas del grupo'
       block_category: Noticias
-  block_group_simple_news:
```

Comment:
Is this block a duplication that needs to be removed? Not sure how it's been used.

Reply:
Yup, while updating the search configuration in that view I noticed there was an unused block. Searched everywhere in our config/code and it wasn't used anywhere.